Task2/Kuba/NoughtsAndCrossesWPF/NoughtsAndCrossesWPF/MainWindow.xaml

```diff
@@ -0,0 +1,25 @@
+﻿<Window x:Class="NoughtsAndCrossesWPF.MainWindow"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:NoughtsAndCrossesWPF="clr-namespace:NoughtsAndCrossesWPF"
+        Title="Noughts And Crosses" Height="380" Width="500">
+        <Window.DataContext>
+            <NoughtsAndCrossesWPF:ViewModel/>
+        </Window.DataContext>
+    <Grid>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="185*"/>
+            <ColumnDefinition Width="307*"/>
+        </Grid.ColumnDefinitions>
```

Comment:
you declared 2 columns, why not 3? you would avoid weird hacks with margins
also, add 3 rows and use them